### PR TITLE
Widen build and build_test dependencies

### PR DESCRIPTION
**Pull request by nshahan**
Update example and add some instructions regarding build_runner.

### CODE REVIEW

File: example/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: sass_builder_example
+version: 0.0.1
```
Comment (kevmoo) [inline example/pubspec.yaml]: Remove the version from the example

**Review by kevmoo — APPROVED** [review]
We need travis on this!

File: example/pubspec.yaml (hunk 1 above)
Comment (nshahan) [reply example/pubspec.yaml]: Done.

**Comment by nshahan** [comment]
Travis testing is the next thing I want to do. I need your input on it though.

**Pull request merged**